Mathlib/Combinatorics/Quiver/Symmetric.lean

```diff
@@ -0,0 +1,247 @@
+/-
+Copyright (c) 2021 David Wärn. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: David Wärn, Antoine Labelle, Rémi Bottinelli
+Ported by: Joël Riou, Rémi Bottinelli
```

Comment:
Have we actually decided to include a `Ported by:` line?

Reply:
No idea, I just followed suit!